spec/models/user_spec.rb

```diff
@@ -2,11 +2,11 @@
 
 RSpec.describe User do
   context "validations" do
-  subject { build(:user) }
-    it  { is_expected.to validate_presence_of(:username) }
-    it  { is_expected.to validate_presence_of(:email) }
-    it  { is_expected.to validate_uniqueness_of(:username) }
-    it  { is_expected.to validate_presence_of(:email) }
-    it  { is_expected.to have_secure_password }
+    subject { build(:user) }
+    it { is_expected.to validate_presence_of(:username) }
```

Comment:
New line between 5-6?

Reply:
Added the line - and picked up an incorrect test, I was testing `validate_presence_of(:email)` twice, instead of the second being for `validate_uniqueness_of(:email`.

Both fixed in sha: 3fded28